UI-Lovelace-Minimalist/UI
# Initial version to support selecting community cards

Related to this discussion: https://github.com/UI-Lovelace-Minimalist/UI/discussions/819

To test it:
- Check `.devcontainer/README.md` getting started steps. 
  - _reopen folder in container_
  - `container start` in terminal

To test upgrade path:
- Make sure you are on main/release. and follow the steps in `.devcontainer/README.md` 
  - _reopen folder in container_
  - `container start` in terminal
- Install the integration
- Stop Home assistant
- Switch to this branch 
- `container start` to start home assistant.
- Hit configure on the integration
- Enable the new community cards option
- Submit
- Now you should see that the integration needs to be reconfigured. (might need a refresh of the page)
- Reconfigure to setup Github Auth. 
- Once that is done you should be able to select cards.

## Screenshots:

![Screenshot from 2022-07-27 22-06-13](https://user-images.githubusercontent.com/10831825/181443482-50b7dcfb-ae27-48c3-bf57-aadc8fd4c8e2.png)
![Screenshot from 2022-07-27 22-06-34](https://user-images.githubusercontent.com/10831825/181443473-d3a43c24-5f7f-404a-9ee1-85cea42a1a1d.png)
![Screenshot from 2022-07-27 22-07-03](https://user-images.githubusercontent.com/10831825/181443474-c5f0ca32-6776-46dc-99b6-b7661a9c1381.png)
![Screenshot from 2022-07-27 22-07-11](https://user-images.githubusercontent.com/10831825/181443477-554553bd-a294-4f92-8696-7143c3ed9136.png)
![Screenshot from 2022-07-27 22-07-42](https://user-images.githubusercontent.com/10831825/181443481-b426782c-1243-484f-9fed-fcb269a103b3.png)
